plutus-ledger-api/src/PlutusLedgerApi/V1/ParamName.hs

```diff
@@ -181,6 +181,7 @@ data ParamName =
   | VerifyEd25519Signature'cpu'arguments'intercept
   | VerifyEd25519Signature'cpu'arguments'slope
   | VerifyEd25519Signature'memory'arguments
-    deriving stock (Eq, Ord, Enum, Ix, Bounded, Generic)
+    -- NB: Show.show != IsParamName.showParamName
```

Comment:
I originally didn't have the Show instance because of this potential confusion, but then I added hunit tests which want to show values when comparing (@=?) . I will fix it with using assertEqual instead.